tool-map.nix

```
@@ -17,7 +17,7 @@ let
              7541f32a4ccca4f97aea3b22f5e593ba2c0267546016b992dfadcd2fe944e55d
           --sha256: sha256-aVI93DtHziicNn2mGli0YE+bC5BeT7mOQQETp2Thi68=
```

While checking out this fix, i got
```bash
nix develop github:input-output-hk/devx/836eaacd6d1c39bf86995882e6fcf4d9a41a67c7#ghc96-windows-iog
evaluation warning: Explicitly set the ABI version of 'webkitgtk'
error: hash mismatch in fixed-output derivation '/nix/store/halmy191ma4yj2rlsy43nm4rv0az8ddr-head.hackage.ghc.haskell.org.drv':
         specified: sha256-aVI93DtHziicNn2mGli0YE+bC5BeT7mOQQETp2Thi68=
            got:    sha256-vciKmt27wOjemc8GhemFdKcN7wIZJGDwUncw9im6PDM=
...
```
So I think this needs to be updated (I tried it as well).
```suggestion
          --sha256: sha256-vciKmt27wOjemc8GhemFdKcN7wIZJGDwUncw9im6PDM=
```

---

Ah, I just saw you comment [here](https://github.com/input-output-hk/devx/issues/195#issuecomment-2657980063)

---

@perturbing thanks! Good catch!